lib/glob.js

```diff
@@ -1,12 +1,19 @@
 'use strict';
 
-var braces = require('braces');
-var brackets = require('expand-brackets');
-var extglob = require('extglob');
-var parse = require('parse-glob');
+var isExtglob = require('is-extglob');
 var chars = require('./chars');
 
 /**
+ * Lazily required module dependencies.
+ */
+
+var lazy = require('lazy-cache')(require);
+var lazyBraces = lazy('braces');
```

Comment:
> the complication didn't seem worth it

Right well if it could/should be done at all, it belongs in `lazy-cache` itself. Over here this makes sense as-is.